fractal_server/app/schemas/workflow.py

```diff
@@ -1,11 +1,35 @@
+from typing import Any
+from typing import Dict
 from typing import List
+from typing import Optional
 
 from sqlmodel import SQLModel
 
 from .task import TaskRead
 
+__all__ = (
+    "WorkflowCreate",
+    "WorkflowRead",
+    "WorkflowUpdate",
+    "WorkflowTaskCreate",
+    "WorkflowTaskRead",
+)
 
-__all__ = ("WorkflowRead", "WorkflowCreate")
+
+class _WorkflowTaskBase(SQLModel):
+    workflow_id: Optional[int]
+    task_id: Optional[int]
+    order: Optional[int]
+    args: Dict[str, Any]
+
+
+class WorkflowTaskCreate(_WorkflowTaskBase):
+    workflow_id: int
```

Comment:
it is better that we leave workflow_id as optional, only task_id is really mandatory.
workflow_id is filled automatically when we add the `WorkflowTask` to a `Workflow.task_list`